src/datascience-ui/data-explorer/reactSlickGridFilterBox.tsx

```diff
@@ -18,20 +33,27 @@ export class ReactSlickGridFilterBox extends React.Component<IFilterProps> {
 
     public render() {
         return (
-            <input
-                type="text"
+            <SearchBox
+                iconProps={filterIcon}
+                onChange={this.updateInputValue}
+                onClear={this.clearInputValue}
                 tabIndex={0}
-                aria-label={this.props.column.name}
+                ariaLabel={this.props.column.name}
```

Comment:
Interesting our linters didn't catch this bug.
How did you come across this?

Reply:
I.e. the change from `aria-label` to `ariaLabel`

Reply:
This file was previously ignored in our eslintrc.js, but this change was made by eslint after I removed this file from the ignore list